src/itemInfo/index.tsx

```diff
@@ -8,15 +8,18 @@ export interface ItemInfoProps {
   readonly icon?: React.ReactNode
   readonly mainTitle?: string
   readonly tag?: JSX.Element
+  readonly ariaLabel?: string
 }
 
-const ItemInfo = ({ mainInfo, className, mainTitle, icon, tag }: ItemInfoProps) => (
+const ItemInfo = ({ mainInfo, className, mainTitle, icon, tag, ariaLabel }: ItemInfoProps) => (
   <Item
     className={className}
     leftTitle={mainTitle}
     leftBody={mainInfo}
     leftAddon={icon}
     tag={tag}
+    ariaLabel={ariaLabel}
+    ariaHidden={!!ariaLabel}
```

Comment:
We should also be able to have an `aria-hidden="false"` with an `aria-label`.
If the `aria-label` is complementary and not a summary.